lms/static/sass/discussion/_layouts.scss

```diff
@@ -1,31 +1,47 @@
 // Layouts for discussion pages
 
-.user-profile {
-    background-color: $sidebar-color;
+.discussion-user-profile-board {
 
-    .user-profile {
-        padding: $baseline;
-        min-height: 500px;
+    .page-title {
+        @include margin-left($baseline);
+        display: inline-block;
     }
 
-    .sidebar-username {
-        font-weight: 700;
-        font-size: $forum-large-font-size;
+    .discussion-profile-info-wrapper {
+        display: inline-block;
     }
 
-    .sidebar-user-roles {
-        margin-top: $baseline/2;
-        font-style: italic;
-        font-size: $forum-base-font-size;
+    .discussion-profile-info  {
+        @include border-right(1px solid $forum-color-border);
+        @include padding(0, $baseline, 0, $baseline);
```

Comment:
Is that he case with biapp rtl rules? I thought that library didn't have shorthand rules...